x-pack/plugins/rule_registry/common/schemas/8.0.0/index.ts

```diff
@@ -35,6 +36,7 @@ const commonAlertIdFieldNames = [ALERT_INSTANCE_ID, ALERT_UUID];
 export type CommonAlertIdFieldName800 = Values<typeof commonAlertIdFieldNames>;
 
 export interface CommonAlertFields800 {
+  [ALERT_RULE_PARAMETERS]: Record<string, unknown>;
```

Comment:
That field is an existing field in the base schema. @elastic/response-ops please advise?

Reply:
It's already defined as a field name (and we use it for Security alerts already) but since it wasn't part of the return from `getCommonAlertFields` we should instead make a new interface `CommonAlertFields870` to represent the updated type. The `index.ts` file in the parent folder would then export `CommonAlertFields870` as `CommonAlertFieldsLatest`.

My hope is in the future when we're reading alerts we'll be able to use these versioned types to read fields from alerts in a type-safe manner, e.g. the types will tell us that `ALERT_RULE_PARAMETERS` has not always existed on alerts but does exist on all alerts for 8.7+, so we have to handle the case where the field is missing on old alerts appropriately but for new alerts we can assume it exists.

Reply:
Thanks for the clarification. I will duplicate the type and version it.